src/core/service.c

```diff
@@ -1263,10 +1262,6 @@ static int service_spawn(
         if (r < 0)
                 return r;
 
-        r = unit_full_printf_strv(UNIT(s), c->argv, &argv);
-        if (r < 0)
```

Comment:
> I am tempted to say that we have to deprecate %c altogether

Well, I don't see any reason to deprecate `%c`. Yes, `Description=%c` is wrong. But `ExecStart=... %c` works fine.